lib/puppet_x/relay/agent/backend/orchestrator.rb

```diff
@@ -45,7 +45,14 @@ def exec(run, _state_dir, schedule)
               message: e.message,
               body: e.response.body,
             })
-            raise
+            run_results = {
+              error: 'Failed to send request to orchestrator API',
+              message: e.message,
```

Comment:
I suspect these two items are superfluous and don't add much to the usable error data that users want to see. I would suggest that we just set `run_results = JSON.parse(e.response.body)` (or perhaps `run_results = {result: JSON.parse(e.response.body)}` for alignment with https://puppet.com/docs/pe/2019.8/orchestrator_api_jobs_endpoint.html#get_jobs_job_id_nodes-results if that makes the most sense).

Reply:
Done